keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -86,7 +89,8 @@ This KEP proposes the new v2 `KeyManagementService` service contract to:
 - enable fully automated key rotation for the latest key
 - improve KMS plugin health check reliability
 - improve observability of envelop operations between kube-apiserver, KMS plugins and KMS
-- if this v2 API reaches GA in release N, the existing v1beta1 gRPC API will be deprecated at release N and removed at release N+3 (the existing key rotation dance of using multiple providers will be used to migrate from v1beta1 to v2)
+- if this v2 API reaches beta in release M, the existing v1beta1 gRPC API will be deprecated at release M
```

Comment:
We deprecated KMSv1 in v1.28 and KMSv2 beta was in v1.27, so should this be deprecated at release M+1?

Reply:
See if you like 5b8d99bd301bff020a4dfe350783b347a0eaef2f better.